build.js

```diff
@@ -1,3 +1,4 @@
+require('svelte/register')
```

Comment:
what does it do?

Reply:
[Render svelte to html](https://svelte.dev/docs#run-time-svelte-register)